### ISSUE TITLE
Node.js  Security WorkGroup Meeting 2020-11-02

### ISSUE THREAD

**Issue by mhdawson** [nodejs/security-wg#692]
## Time

**UTC Mon 02-Nov-2020 13:00 (01:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Mon 02-Nov-2020 05:00 (05:00 AM) |
| US / Mountain | Mon 02-Nov-2020 06:00 (06:00 AM) |
| US / Central  | Mon 02-Nov-2020 07:00 (07:00 AM) |
| US / Eastern  | Mon 02-Nov-2020 08:00 (08:00 AM) |
| EU / Western  | Mon 02-Nov-2020 13:00 (01:00 PM) |
| EU / Central  | Mon 02-Nov-2020 14:00 (02:00 PM) |
| EU / Eastern  | Mon 02-Nov-2020 15:00 (03:00 PM) |
| Moscow        | Mon 02-Nov-2020 16:00 (04:00 PM) |
| Chennai       | Mon 02-Nov-2020 18:30 (06:30 PM) |
| Hangzhou      | Mon 02-Nov-2020 21:00 (09:00 PM) |
| Tokyo         | Mon 02-Nov-2020 22:00 (10:00 PM) |
| Sydney        | Tue 03-Nov-2020 00:00 (12:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Security%20WorkGroup+Meeting+2020-11-02&iso=20201102T1300
* or https://www.wolframalpha.com/input/?i=01PM+UTC%2C+Nov+02%2C+2020+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1IVv76TNYhZStATlPlpkJYic1fu-WfR0t5iiihwsQCn8/edit>

## Agenda

Extracted from **security-wg-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/security-wg

* Nominate GENTILHOMME Thomas as member of the WG [#678](https://github.com/nodejs/security-wg/issues/678)
* Refocusing the Ecosystem Security WG on OpenJS Foundation projects [#662](https://github.com/nodejs/security-wg/issues/662)
* Adding myself to the Members List based on #434  [#627](https://github.com/nodejs/security-wg/pull/627)

## Invited

* Security wg team: @nodejs/security-wg

## Notes

The agenda comes from issues labelled with `security-wg-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/405474359
* For those who just want to watch We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up. 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

**Comment by lirantal** [comment]
Connecting from mobile cause my laptop broke but looks like the meeting didn't start? /cc @mhdawson

**Comment by fraxken** [comment]
Also waiting. Maybe not the best schedule because i guess the NodeConf is at the same time ?

**Comment by esarafianou** [comment]
It's quite unfortunate that we're not meeting the past few months especially considering that we'd like to refocus the Ecosystem Security WG on OpenJS Foundation projects (#662)

@mhdawson is there a better time for you that you could join the meeting? Otherwise, could we consider moving the meeting to a new zoom link? I'd happily volunteer being the Host making sure I always join the meeting and allow the rest of participants in the zoom link.

**Comment by MarcinHoppe** [comment]
I think today we might have clashed with [Node Conf Remote](https://www.nodeconfremote.com).

**Comment by lirantal** [comment]
Yep, must've been the NodeConf take over ;-)
Can we perhaps push this one for next week Monday as a one-off just to make sure we don't wait another month?

**Comment by mhdawson** [comment]
@esarafianou it is too early for me (although today it also conflicted with NodeConf.eu). We had Sam setup to host/stream and we can get access for others to start that zoom/stream.  @esarafianou if you are volunteering to do that I can help get you access to the Node.js Zoom, Youtube accounts so you could kickoff the meetings.

**Comment by esarafianou** [comment]
@mhdawson Yes, I can volunteer kicking off the meetings. Let's sync to get me the necessary access.

**Comment by mhdawson** [comment]
@esarafianou I've started the process, first asking the tsc/commcom for the ok to get you access. Once I have the ok, I'll cc on emails related to the next steps.
